src/main/java/core/basesyntax/models/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class Rectangle implements Figure {
+    private final String color;
+    private final double width;
+    private final double height;
+
+    public Rectangle(String color, double width, double height) {
+        this.color = color;
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+     public void draw() {
+        System.out.println("Color: " + color
+                + "Type: Rectangle "
+                + "Width: " + width
+                + "Height: " + height);

```

Comment:
In the `draw` method, the 'Type: Rectangle' string is concatenated without a space before it, which will lead to a printing error. Consider adding a space before 'Type' for correct output formatting.